tox.ini

```diff
@@ -25,3 +27,5 @@ deps =
     flake8
     -rdev-requirements.txt
 commands=flake8 script_runner
+
+[testenv:pywin]
```

Comment:
Yes, I have found no other way of running just the default [testenv]
environment. The goal here is to run only the tests, not the docs
(especially since the docs need make)

It might be also useful if there were any windows-specific deps, but at
this point there aren't any :)

On Fri, Mar 25, 2016 at 11:38 AM, Dan Kilman <notifications@github.com>
wrote:

> In tox.ini
> <https://github.com/cloudify-cosmo/cloudify-script-plugin/pull/36#discussion_r57434616>
> :
>
> > @@ -25,3 +27,5 @@ deps =
> >      flake8
> >      -rdev-requirements.txt
> >  commands=flake8 script_runner
> > +
> > +[testenv:pywin]
>
> when you specify a configuration on tox like this, does it mean it
> inherits all configuration from testenv?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cloudify-cosmo/cloudify-script-plugin/pull/36/files/d8bd7dd2f4f4109d9c32a0b04da5d861852962d9#r57434616>
>



-- 
Łukasz Maksymczuk
Software Engineer


---------------------------------
M: +48 881 616 544
E: lukasz.maksymczuk@codilime.com
---------------------------------

CodiLime Sp. z o.o. - Ltd. company with its registered office in Poland,
01-167 Warsaw, ul. Zawiszy 14/97. Registered by The District Court for the
Capital City of Warsaw, XII Commercial Department of the National Court
Register. Entered into National Court Register under No. KRS 0000388871.
Tax identification number (NIP) 5272657478. Statistical number (REGON)
142974628.

-----------------------------------------

The information in this email is confidential and may be legally
privileged, it may contain information that is confidential in CodiLime Sp.
z o.o. It is intended solely for the addressee. Any access to this email by
third parties is unauthorized. If you are not the intended recipient of
this message, any disclosure, copying, distribution or any action
undertaken or neglected in reliance thereon is prohibited and may result in
your liability for damages.